.github/workflows/pythonapp.yml

```diff
@@ -64,10 +99,15 @@ jobs:
         # Use always() to always run this step to publish test results
         # when there are test failures
         if: always()
+
+      - name: Setup cl.exe (Windows)
```

Comment:
Is `cl.exe` clang?

Reply:
No, it's Microsoft's Visual C Compiler (MSVC).

Note that Microsoft also offer `clang-cl.exe` which is a wrapper over `clang` to a provide drop-in replacement for `cl.exe`. This looked exciting in the first instance, because it supports `_Complex`. However, `clang-cl.exe` is not supported by Conda or CFFI, where MSVC is the only supported option today on Windows.

I did however add a test of `clang-cl.exe` to our demo pipeline, and it works fine.